tests/custom/test_client.py

```
@@ -1,7 +1,49 @@
+async def test_async_pipedream_async_raw_access_token() -> None:
+    """Test AsyncPipedream async method async_raw_access_token() with access_token."""
+    client = AsyncPipedream(access_token="test-token", project_id="test-project")
+    token = await client.async_raw_access_token
+    assert token == "test-token"
+
+
+async def test_async_pipedream_async_raw_access_token_with_oauth() -> None:
+    """Test AsyncPipedream async method with OAuth flow."""
+    client = AsyncPipedream(
+        client_id="test-client-id",
+        client_secret="test-client-secret",
```

Ruff (0.14.8)

Possible hardcoded password assigned to argument: "client_secret"

(S106)